ProcessMaker/bpm
# Add the node script executor

Fixes #1547 
Depends on https://github.com/ProcessMaker/pm4-docker-executor-node being pushed to dockerhub, but can be tested with the below instructions

This PR adds the node executor with SDK

To test, run commands in the vagrant box:
- Clone the executor to a folder on vagrant:
  `git clone https://github.com/ProcessMaker/pm4-docker-executor-node.git ~/pm4-docker-executor-node.git`
- Run the build script `./build.sh`
- In bpm, edit your .env and add
  `SCRIPTS_NODE_IMAGE=processmaker/pm4-docker-executor-node:dev-master`
- Import the process tests/Manual/PhpSDKScripts/node_sdk_process.bpm4
- Change the script user to admin
- Run the process "Test Node Executor"
- When it finishes, verify that the data contains a list of all users' emails
